compiler/rustc_hir_typeck/src/method/suggest.rs

```diff
@@ -3228,6 +3229,61 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         }
     }
 
+    fn suggest_bounds_for_range_to_method(
+        &self,
```

Comment:
This should only suggest `0` if the element type is integral.

Right now, you  suggest `(0..'f').rev()` for `(..'f').rev()`. This affects all types that implement the `Step` trait and aren't integral.

Reply:
Also, should it be 0 if the element is signed?